tests/js/instanceof.js

```diff
@@ -20,43 +20,43 @@ suite('Built-in Element instanceof', function() {
     // prototype chain to make instanceof work.
     var elements = [
       'a',
-      // 'abbr',
-      // 'address',
+      'abbr',
+      'address',
       'area',
-      // 'article',
-      // 'aside',
+      'article',
+      'aside',
       'audio',
-      // 'b',
+      'b',
       'base',
-      // 'bdi',
-      // 'bdo',
-      // 'blockquote', // fails on IE11
+      'bdi',
+      'bdo',
+      'blockquote', // fails on IE11
```

Comment:
presumably these "fails" comments can be removed, assuming they were due to differences in constructors between browsers?